layouts/partials/head.html

```diff
@@ -17,7 +17,11 @@
   <link rel="icon" href="{{ `images/favicon.png` | absURL }}" type="image/x-icon">
 
   {{ "<!-- fonts -->" | safeHTML }}
-  <link href="https://fonts.googleapis.com/css?family={{replace .Site.Params.font_family ' ' '+' | title}}:300,400,700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family={{replace .Site.Params.font_family_a ' ' '+' | title}}+{{replace .Site.Params.font_family_b ' ' '+' | title}}:wght@300;400;700&display=swap" rel="stylesheet"> 
```

Comment:
Hmm, I am curious. Why was this change necessary? Not sure I follow your path of thinking.

Reply:
Remember 

> Hi @Idadelveloper, this looks great and I am almost ready to merge this! The only thing that is a red flag to me for this Pull Request is the way fonts are loaded into the theme from the config. It is possible there is a bug here you caught working on this, but if it is a bug, we should look at fixing this another way that does not depend on the config file values.
> 
> Does this make sense? Let me know if you have doubts. Thanks again for working on this! 🙌🏻

@jwflory  there's definitely a bug. Will get rid of this change together with the changes in the config file.